sunpy/instr/lyra.py

```diff
@@ -450,9 +451,9 @@ def get_lytaf_events(start_time, end_time, lytaf_path=None,
     for suffix in combine_files:
         # Check database files are present
         dbname = "annotation_{0}.db".format(suffix)
-        check_download_file(dbname, LYTAF_REMOTE_PATH, lytaf_path)
```

Comment:
Deprecate `check_download_file` @Cadair @nabobalis

Reply:
Yes? We don't use it anymore?

Reply:
I think we can probably do that I know @kakirastern was using it in IRISPy over the summer though. Makes me wonder if it's being used anywhere else. Maybe we should do an extended deprecation on it?

Reply:
Make it so.